_posts/2023-11-24-this-week-in-rails.markdown

```diff
@@ -3,14 +3,15 @@ layout: post
 title: "ErrorReporter#unexpected, with_routing test helpers and more!"
 categories: news
 author: vipulnsward
+og_image: assets/images/this-week-in-rails.png
 published: true
 date: 2023-11-24
 ---
 
 
 Happy Thanksgiving weekend everyone! This is [Vipul](https://www.saeloun.com/team/vipul) bringing you the latest from this week's changes in the Rails codebase.
 
-[ErrorReporter#unexpected to report in production but raise in development](https://github.com/rails/rails/pull/49951)  
```

Comment:
@gregmolnar FYI, I think your editor auto-removed the trailing spaces on this post on save, so the line breaks might be messed up here.

Reply:
Bummer. I will fix it when doing the year end issue tomorrow.